packages/server/src/threads/automation.ts

```diff
@@ -241,7 +241,7 @@ class Orchestrator {
     })
   }
 
-  async execute() {
+  async execute(): Promise<any> {
```

Comment:
Again this would be great to type - but the way it works today its very hard to understand all of the returns - probably suggests it needs a bit of refactoring here.

Reply:
I did attempt both this one and the one above but they can return wildly different things, which the callsites don't seem to care about, they just bung the result into a `ctx.body` and call it a day.

Something we should certainly address at some point.